xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -759,12 +759,15 @@ void CGUIDialogPVRTimerSettings::InitializeTypesList()
     if (type->ForbidsEpgTagOnCreate() && m_timerInfoTag->GetEpgInfoTag())
       continue;
 
-    EPG::CEpgInfoTagPtr epgTag = m_timerInfoTag->GetEpgInfoTag(false);
-    //If we have an epg tag, ask the pvr addon if recording is possible, else, use the current time
-    bool canRecord = epgTag ? epgTag->IsRecordable() : m_timerInfoTag->EndAsLocalTime() >= CDateTime::GetCurrentDateTime();
-    // Drop TimerTypes that aren't rules or cannot be recorded
-    if (!type->IsTimerRule() && !canRecord)
-      continue;
+    // Drop TimerTypes that aren't rules and cannot be recorded
+    if (!type->IsTimerRule())
+    {
+      EPG::CEpgInfoTagPtr epgTag = m_timerInfoTag->GetEpgInfoTag(false);
+      //If we have an epg tag, ask the pvr addon if recording is possible, else, use the current time
```

Comment:
Now we're getting to the nitpicking part: always put a space between // and the actual comment. Kodi coding convention. Thanks.